c++/test/CreateTestFiles.cc

```diff
@@ -31,8 +31,8 @@
 void writeCustomOrcFile(const std::string& filename,
                         const orc::proto::Metadata& metadata,
                         const orc::proto::Footer& footer,
-                        const std::vector<uint>& version,
-                        uint writerVersion) {
+                        const std::vector<unsigned>& version,
```

Comment:
We generally prefer to use explicitly sized types, so uint64_t or uint32_t would be better.